static/main.js

```diff
@@ -0,0 +1,59 @@
+$(document).ready(function(){
+    $(".hide-on-start").hide();
+
+    $('#edit-link').on('click', function(event){
+        event.preventDefault();
+
+        var id = $('#form-id').val();
+        var title = $('#form-title').val();
+
+        $.ajax({
+            method: "GET",
+            url: "/edit/" + id + "/" + title
+        }).done(function(response){
+            $("#form-title").val();
+            $('#form-text').val();
+            $('#twitter').hide();
+            $('.entry').hide();
+            $('#edit-form-container').show();
+        }).fail(function(){
+            alert("error");
```

Comment:
eventually, you'll want to convert this to a more user-friendly error reporting method.  You could put a hidden 'error' div in your base template and then fill and display it with information received back in the case of an exception.  The function here takes a few arguments, which include the error itself, and reports about the message it contains.